Documentation/Diagnostics/PH2146.md

```diff
@@ -0,0 +1,59 @@
+# PH2146: Avoid calling ToString() on string type
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2146 |
+| Category  | [Maintainability](../Maintainability.md) |
+| Analyzer | [AvoidToStringOnStringAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/main/Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidToStringOnStringAnalyzer.cs)
+| CodeFix  | Yes |
+| Severity | Info |
+| Enabled By Default | Yes |
```

Comment:
Let's changed Enabled by default (in code and docs) to No. (until a CodeFixer can be created, and tested on a production environment)